src/plugins/navigation/public/plugin.tsx

```diff
@@ -195,7 +195,7 @@ export class NavigationPublicPlugin
       }
     }
 
-    if (isProjectNav) {
+    if (isProjectNav && solutionView !== 'classic') {
```

Comment:
is this to make typescript happy? `isProjectNav` will always be true when `solutionView !== 'classic'`, i guess unless we introduce a different solutionView value in the future.

Reply:
Yep, just to make TS happy 😊